python/README.md

```diff
@@ -1,15 +1,34 @@
-# Python Examples
-Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics
+# kafka-python Examples
+Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics. This works with the Apache Kafka Wire level Protocol. 
+This module contains Python producer and consumer programs.These programs accept the following 2 arguments:
+1.Bootstrap server and the port: <DataAccessGateway ip address or hostname: port> 
```

Comment:
Please put a space after the period so markdown works right

Reply:
Sure Ted. Will do